src/commands/githubActions/bumpUpstream/setup.ts

```diff
@@ -0,0 +1,99 @@
+import { Manifest, Compose } from "@dappnode/types";
```

Comment:
This file is confusing, has arbitrary functions in it, either find a better file struct or create the required modules to achieve a better organization